src/backend/gog/games.ts

```diff
@@ -845,6 +849,14 @@ class GOGGame extends Game {
     const gameObject = installedArray[gameIndex]
 
     if (gameData.install.platform !== 'linux') {
+      // Force getting new cache
+      if (
+        gogInstallInfoStore.has(`${this.appName}_${gameData.install.platform}`)
+      ) {
+        gogInstallInfoStore.delete(
+          `${this.appName}_${gameData.install.platform}`
+        )
+      }
```

Comment:
You could just call `.delete` directly here, it'll do nothing if the key isn't in the store